src/components/player-play-pause-button/player-play-pause-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { Fragment } from 'react';
+
+type PlayerPlayPauseButtontType = {
+  handleClick: () => void;
+  isPlaying: boolean;
+};
+
+export default function PlayerPlayPauseButton({
+  handleClick,
+  isPlaying,
+}: PlayerPlayPauseButtontType): JSX.Element {
+  return (
+    <button onClick={handleClick} type="button" className="player__play">
+      {isPlaying ? (
+        <Fragment>
```

Comment:
Если key не нужен то используй короткую запись
```suggestion
        <>
```